File: beecrowd/1160.py
t = int(input())                     #casos de teste
contanos = []                        #lista que guarda a contagem dos anos
for c in range(t):
    cont = 0
    dadospop = input().split()
    pa = int(dadospop[0])
    pb = int(dadospop[1])
    g1 = float(dadospop[2])
    g2 = float(dadospop[3])
    while True:
        if pa > pb:
            break
        if cont > 100:
            break
        cont += 1
        cpaa = int((pa * g1) / 100)  #crescimento populacional absoluto da cidade A
        cpab = int((pb * g2) / 100)  #crescimento populacional absoluto da cidade B
        pa += cpaa
        pb += cpab
    contanos.append(cont)
for h in range(t):
    if contanos[h] > 100:
        print('Mais de 1 seculo.')
    else:
        print(f'{contanos[h]} anos.')

File: beecrowd/1160.c
#include <stdio.h>

int main() {
    int testes, pa, pb, contador; //pa = pop de A, pb = pop de b 
    float g1, g2, cpaa, cpab;     //g1 = taxa de cresc de A, g2 = taxa de cresc de B

// cpaa = cresc populacional absoluto de A, cpab = cresc pop abs de B
    scanf("%d", &testes);

    while(testes) {
        testes--;
        contador = 0;
        scanf("%d %d %f %f", &pa, &pb, &g1, &g2);
        while(1) {
            if(pa > pb || contador > 100) {
                break;
            }
            contador++;
            int(cpaa) = (pa * g1) / 100;
            int(cpab) = (pb * g2) / 100;
            pa += cpaa;
            pb += cpab;
        }
        if(contador > 100) {
            printf("Mais de 1 seculo.\n");
        }else {
            printf("%d anos.\n", contador);
        }
    }

    return 0;
}